fast/stages/2-networking-c-nva/README.md

```diff
@@ -64,23 +73,36 @@ The "landing zone" is divided into two VPC networks:
 - the trusted VPC: the connectivity hub towards other trusted networks
 - the untrusted VPC: the connectivity hub towards any other untrusted network
 
-The VPCs are connected with two sets of sample NVA machines, grouped in regional (multi-zone) [Managed Instance Groups (MIGs)](https://cloud.google.com/compute/docs/instance-groups). The appliances are plain Linux machines, performing simple routing/natting, leveraging some standard Linux features, such as *ip route* or *iptables*. The appliances are suited for demo purposes only and they should be replaced with enterprise-grade solutions before moving to production.
-The traffic destined to the VMs in each MIG is mediated through regional internal load balancers, both in the trusted and in the untrusted networks.
+### NCC, NVAs and BGP sessions
+
+The VPCs connect through two sets of sample NVA machines: one per region, two VMs per region. The appliances run [Contrainer-Optimized OS](https://cloud.google.com/container-optimized-os/docs) and a container with [FRRouting](https://frrouting.org/).
```

Comment:
Nit: rephrase

The VPCs connect through two sets of sample NVA machines: one per region, each containing two instances.